apps/src/dance/ai/types.ts

```diff
@@ -30,3 +30,36 @@ export interface AiFieldValue extends GeneratedEffect {
 }
 
 export type DanceAiSound = (typeof DANCE_AI_SOUNDS)[number];
+
+export type EmojiItem = {
+  id: string;
+  emoji: string;
+};
+
+export type AIEmojiItem = EmojiItem & {
+  modelDescriptiveName: string;
```

Comment:
Should we keep the first 'i' in 'AIEmojiItem' lowercase for consistency?

Reply:
💯 Just a typo oversight. Fixed!